### ISSUE TITLE
Making the espeak tr voice read the cyrillic letters using russian language

### ISSUE THREAD

**Issue by fatihyuksek1** [espeak-ng/espeak-ng#1652]
I want to be able to use the Turkish and Russian languages together, so I want the Turkish voice to read the Cyrilic letters as in Russian.
How do I go about doing this?

**Comment by jaacoppi** [comment]
> I want to be able to use the Turkish and Russian languages together, so I want the Turkish voice to read the Cyrilic letters as in Russian.

This will not be easy. You could start by checking out how russian handles arabic letters. Then implement the opposite in Turkish.

**Comment by fatihyuksek1** [comment]
The Turkish language uses the Latin alphabet, same as English. What does it have to do with Arabic letters, if I may ask? От: Juho HiltunenОтправлено: 11 февраля 2023 г. в 10:52Кому: espeak-ng/espeak-ngКопия: Fatih Yüksek; AuthorТема: Re: [espeak-ng/espeak-ng] Making the espeak tr voice read the cyrillic letters using russian language (Issue #1652) > I want to be able to use the Turkish and Russian languages together, so I want the Turkish voice to read the Cyrilic letters as in Russian.This will not be easy. You could start by checking out how russian handles arabic letters. Then implement the opposite in Turkish.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by fatihyuksek1** [comment]
I wrote a reply using the windows 10 Mail app, but the reply seems to be sent with the html code, so this is the text of the reply, received from the html code:
The Turkish language uses the Latin alphabet, same as English. What does it have to do with Arabic letters, if I may ask?

**Comment by jaacoppi** [comment]
Sorry, I meant cyrillic letters, not arabic.

Also, you'll want to understand "_^_" from docs/dictionary.md. Mandarin chinese (cmn) uses that a lot to default latin characters to English. You could use a similar solution to default cyrillic characters to Turkish.

**Comment by fatihyuksek1** [comment]
No, I want to use a method where when espeak Turkish sees a Cyrillic letter, it should switch to Russian language, like the Georgian voice does, for example.

**Comment by fatihyuksek1** [comment]
I totally agree with you 100%.6 апр. 2023 г., в 10:04, Vsevolod Popov ***@***.***> написал(а):﻿
I think it would be useful to have a possibility to choose prefered language for latin characters for languages that not use latin characters as their base. For example for Russian, arabic, it would be useful to be able to change prefered language for Latin characters if that it is even possible. For example, I'd like to read Russian and Polish with eSpeak together or Russian and Spanish. I suppose it is not possible to implement all that, but who knows why not to write about that?
